### ISSUE TITLE
Tensorboard logging 

### ISSUE THREAD

**Issue by JonasEinig** [spaethju/skin-cancer-detection#7]
log several values to tensorboard to be able to debug the network:
loss
logits
etc.

**Comment by JonasEinig** [comment]
c00048de3943ad5ca074f8ce3425feb235ea943b
